dbms/src/Storages/DeltaMerge/Remote/DataStore/DataStoreS3.cpp

```diff
@@ -103,6 +133,7 @@ bool DataStoreS3::putCheckpointFiles(
 
     std::vector<std::future<void>> upload_results;
     // upload in parallel
+    // Note: Local checkpoint files are always not encrypted.
```

Comment:
We will not write checkpoint files in the local disk after https://github.com/pingcap/tiflash/pull/8385 merged.